Added GSoC estefania's report blog post

ojeda-e opened the pull request:
Hi @hmacdope, @orbeckst 
This blog post will help with #193 and closes #192.

Suggestions are welcome! 
I'm tagging @lilyminium too just because the curvature plots have the same color bar for both subplots and she said it would be nice ☺️ .

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+- Data visualization made easy.
+
+<img
+src="https://ojeda-e.github.io/assets/images/patches.png"
```
add the image to [public/images](https://github.com/MDAnalysis/MDAnalysis.github.io/tree/master/public/images) (you can also make a subdirectory) please — better to have images locally hosted

orbeckst replied:
Then see https://github.com/MDAnalysis/MDAnalysis.github.io#images

orbeckst replied:
Given that you have multiple images, make a subdirectory, please!

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+calculate mean and Gaussian curvature. After refactoring ([#34]), the core functions 
+of MembraneCurvature were cleaned and tuned up ([#40] and [#44]). Our core functions are: 
+
+- [`derive_surface`], [`get_z_surface`], [`normalize_grid`] ([#40])
+- [`mean_curvature`], [`gaussian_curvature`]. ([#44])
```
Instead of linking to PRs, link to your code docs. That will be more interesting for readers.

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+
+
+## Analysis Base ([#43], [#48])
+The analysis in MembraneCurvature used the [MDAnalysis AnalysisBase](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/base.html) building block, from where we obtained the basic structure to run multilframe analysis. 
```
write this link as 
```suggestion
The analysis in MembraneCurvature used the [MDAnalysis AnalysisBase]({{ site.docs.mdanalysis.url }}/stable/documentation_pages/analysis/base.html) building block, from where we obtained the basic structure to run multilframe analysis. 
```
using site variables from [_config.yml](https://github.com/MDAnalysis/MDAnalysis.github.io/blob/master/_config.yml)

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+## Analysis Base ([#43], [#48])
+The analysis in MembraneCurvature used the [MDAnalysis AnalysisBase](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/base.html) building block, from where we obtained the basic structure to run multilframe analysis. 
+
+In the [MembraneCurvature subclass](https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/base.py#L27) of AnalysisBase,
```
Also have a link to the docs

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+MembraneCurvature MDAnalysis tool: [Core functions](#core-functions-34-40-44),
+[AnalysisBase](#analysis-base-43-48), and [Documentation](#documentation-57-62-64-69).
+
+## Core functions ([#34], [#40], [#44])
```
Did you test that `[#34]` resolves as a link? Maybe `[#34][]` does the job?

ojeda-e replied:
Hmmm... I tested all the PR links, indeed. It is slow but links accordingly.

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+
+
+# How can I use it?
+Membrane-curvature uses [MDAnalysis] under the hood. We can install
```
link to your installation instructions, too

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+<span style="color:#A23333">POPC:POPE</span><span style="color:grey">:CHOL</span>, in a 
+<span style="color:#A23333">5:4</span><span style="color:grey">:1</span> ratio:
+
+<img src="https://ojeda-e.github.io/assets/images/movie_red.gif" width="800" height="400" /> 
```
local image

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
````diff
@@ -0,0 +1,299 @@
+```
+
+## Plots
+To visualize the results from `MembraneCurvature.run()`, we can use [contourf] from Matplotlib. 
````
link to docs?

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+To visualize the results from `MembraneCurvature.run()`, we can use [contourf] from Matplotlib. 
+For example, here is the plot of the averaged results for the upper leaflet using contours:
+
+<img src = "https://ojeda-e.github.io/assets/images/results_dark.png" alt = "results"
```
local image

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
````diff
@@ -0,0 +1,299 @@
+cbar.set_label('H (nm$^{-1}$)', fontsize=6, labelpad=2) 
+```
+
+<img src = "https://ojeda-e.github.io/assets/images/mycontours_dark.png" 
````
local image

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+This would definitely be a nice improvement for a future release of
+MembraneCurvature!
+
+![vesicles](https://ojeda-e.github.io/assets/images/vesicles.png)
```
local image

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+Participating in GSoC with MDAnalysis has been a unique experience. I had the
+opportunity to learn best practices in software development mentored by a group
+of incredibly talented people:
+- [@lilywang](https://github.com/lilyminium) (Lily),
```
just use the GH handle and it will automatically resolve, i.e.
```markdown
@lilyminium (Lily), @IAlibay (Irfan), ...
```
and note that Lily's handle is `@lilyminium` !

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+- [@fiona-naughton](https://github.com/fiona-naughton) (Fiona), and
+- [@orbeckst](https://github.com/orbeckst) (Oliver). 
+
+I also would like to thank [@richardjgowers](https://github.com/richardjgowers) (Richard) and 
+[@tylerjereddy](https://github.com/tylerjereddy) (Tyler) from the MDA community, who participated in our
```
same, don't add URLs fior GH handles

orbeckst commented on _posts/2021-09-02-gsoc-final-report-estefania.md:
```diff
@@ -0,0 +1,299 @@
+[#66]: https://github.com/MDAnalysis/membrane-curvature/pull/66
+[#69]: https://github.com/MDAnalysis/membrane-curvature/pull/69
+
+[`mean_curvature`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/curvature.py#L67
+[`gaussian_curvature`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/curvature.py#L39
+[`normalize_grid`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/surface.py#L108
+[`get_z_surface`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/surface.py#L52
+[`derive_surface`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/surface.py#L22
```
I'd make these links to your docs and I'd also add parentheses after functions, eg `mean_curvature()`.

orbeckst reviewed (changes requested):
Very nice. Apart from minor things, I'd like to have the images locally on the GitHub webserver instead of pulling them from your blog assets. 

I'd also suggest you think a bit more about the audience: The report blog post told your evaluators what you accomplished so you referenced PRs extensively. For _this_ post I'd consider the audience potential users, who likely care more about how to use your tool and what they can do with it (than admiring the hard work you did). You wou;dn't have to change much, just use more links to your docs and less focus on "code written", more on "things that can be accomplished".

ojeda-e commented:
Thanks for the comments and suggestions, @orbeckst. These are the changes from the last push:

- [x] Added folder `public/images/final_report_estefania/` and saved the respective `.png` and `.gif` files.
- [x] Added summary section with a figure.
- [x] Removed most of the links to PRs. Left mainly the ones in the headers.
- [x] Fixed GH usernames and links. (Apologies to @lilyminium)
- [x] Changed colors of figures. (From green to orange to match MDAnalysis theme ☺️  )
- [x] Added links to docs instead of PRs when needed.

I think the blog post looks nicer after these improvements. Please me know I miss anything here. Thanks.

hmacdope commented:
@ojeda-e reads really well to me! 👍 

Have your points been addressed @orbeckst?

orbeckst commented:
Very nice!

Sorry @ojeda-e and @hmacdope , this notification slipped through, it's been busy days.
